astrogrism/grism_observation.py

```diff
@@ -73,6 +86,27 @@ def __init__(self, grism_image, direct_image=None, telescope=None, instrument=No
         else:
             self.geometric_transforms = self._build_geometric_transforms()
 
+    def _calculate_subarray_offsets(self, instrument):
+
+        image_header = self.grism_image["SCI"].header
+        centera1 = image_header["CENTERA1"]
+        centera2 = image_header["CENTERA2"]
+        sizaxis1 = image_header["SIZAXIS1"]
+        sizaxis2 = image_header["SIZAXIS2"]
+
+        if instrument == "WFC3_UVIS":
+            x_offset = centera1 - (sizaxis1 / 2) - 1
+            # This accounts for serial_over from wf3tools.sub2full
+            y_offset = centera2 - (sizaxis2 / 2) - 26
+        elif instrument == "WFC3_IR":
+            x_offset = centera1 - (sizaxis1 / 2) - 1
+            y_offset = centera2 - (sizaxis2 / 2) - 1
+        else:
+            raise ValueError(f"Subarrays not currently supported for {instrument}")
+
+        print(x_offset, y_offset)
```

Comment:
Is this print statement intended to be left in?

Reply:
Nope, good catch. Was just for testing during development.